.ci/snapshot.sh

```diff
@@ -34,3 +34,6 @@ fi
 
 echo "--- Deploy the snapshot :package: [./mvnw $GOAL)] $DRY_RUN_MSG"
 ./mvnw -V -s .ci/settings.xml -Pgpg clean $GOAL -DskipTests --batch-mode | tee snapshot.txt
+
+echo "--- Archive the target folder with jar files"
+find . -type d -name target -exec find {} -name '*.jar' -print0 \; | xargs -0 tar -cvf "${TARBALL_FILE:-dist.tar}"
```

Comment:
Use `.ci/scripts/published-artifacts-list.sh` script added in #3595 instead to get the list of published artifacts

Reply:
see https://github.com/elastic/apm-agent-java/pull/3594/commits/ce87909b089a09acaa879bfc8fe73eae3ad890ca